tests/e2e/expected_results/urllib3/urllib3_cue_invalid.json

```diff
@@ -352,7 +352,7 @@
                     ],
                     "justification": [
                         "Successfully verified level 3: ",
-                        "verify passed: urllib3-2.0.3-py3-none-any.whl,verify passed: urllib3-2.0.3.tar.gz"
+                        "verify passed : urllib3-2.0.3-py3-none-any.whl,verify passed : urllib3-2.0.3.tar.gz"
```

Comment:
Is the addition of the spaces here expected?

Reply:
This is due to the space added [here](https://github.com/oracle/macaron/blob/d6f1cc60a5eb1c26165f530afc73ea735e800e38/src/macaron/slsa_analyzer/checks/provenance_l3_check.py#L71) and is out of the scope to be fixed in this PR.

See this issue: https://github.com/oracle/macaron/issues/380